samples/gazell/gzll_ack_payload/device/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: Gazell ACK payload device sample
+tests:
+  samples.gazell.gzll_ack_payload.device.build:
+    build_only: true
+    build_on_all: true
+    platform_allow: nrf52dk_nrf52832 nrf52833dk_nrf52833 nrf52840dk_nrf52840
```

Comment:
Add `integration_platforms`. It can have the same platforms as in `platform_allow`

Reply:
Will do.